examples/05_nossr/src/pages/about.tsx

```diff
@@ -28,5 +28,5 @@ const getData = async () => {
 export const getConfig = async () => {
   return {
     render: 'static',
-  };
+  } as const;
```

Comment:
This is needed to tell typescript that the type is `{ render: 'static' }` and otherwise it will be `{ render: string }` which will not work

Reply:
This is very unfortunate.

Reply:
I agree... importing from this file to the plugin would be a way to get the literal value into the generation, but I can't think of other ways quite yet.

Reply:
one other way would be to add `satisfies GetConfigResult` instead of `as const` or something like this, but that would probably be less desirable given that it's longer.

Reply:
`as const` is better.